packages/shared/src/components/post/PostComments.tsx

```diff
@@ -73,6 +73,7 @@ export function PostComments({
       {
         enabled: !!id && tokenRefreshed,
         refetchInterval: 60 * 1000,
+        refetchOnWindowFocus: false,
```

Comment:
Yeah thought about it, but do we really care that much?
It's mainly a issue on companion maybe we could only disable it there eventually?

Reply:
Is it possible to tell it to only refetch on focus if the current cache is stale?

Reply:
You want the refresh more from perspective of new comments could have been loaded, something react query doesn't know at that point.
So it's more fire in hopes of there being something for now.

Reply:
Don't care that much for now

Reply:
> Is it possible to tell it to only refetch on focus if the current cache is stale?

This is the behaviour but all of our queries are stale by default because we don't set the `staleTime`.

Reply:
We talked before that in the future we should define `staleTime` + invalidation scenarios for all our queries which will allow us to have optimal amount of refetches.